server/src/main/java/com/exacaster/lighter/application/sessions/SessionService.java

```diff
@@ -95,10 +113,14 @@ public Optional<Application> fetchOne(String id) {
     }
 
     public void deleteOne(String id) {
-        this.fetchOne(id).ifPresent(this::deleteOne);
+        fetchByType(id, ApplicationType.SESSION).ifPresent(this::deleteOne);
```

Comment:
do we wanna  allow SessionService to remove all kind of application or just session ?

Reply:
It should remove only sessions.

Reply:
done.